frontend/mobilityai_react/App/Screens/PatientData.js

```diff
@@ -43,62 +40,77 @@ export default class PatientData extends Component {
             lastName: props.lastName,
             barColour: arrayColours['unknown'],
             data: [0,0,0,0,0,0,0,0,0,0,0,0,0],
+            date: props.date,
             movementPercentages: {'sitting': {total: 0, bar: new Array(13)}, 'standing': {total: 0, bar: new Array(13)}, 'lyingDown': {total: 0, bar: new Array(13)}, 'walking': {total: 0, bar: new Array(13)}, 'unknown': {total: 0, bar: new Array(13)}},
         }
-        // this.onPress = this.onPress.bind(this);
-        //this.props.navigate = props.navigate;
     };
 
     _onPressButton(activityColour, newData) {
         this.setState({ barColour: arrayColours[activityColour]});
         this.setState({ data: newData });
     };
 
-    // TODO: Have Nav Header display patient name
-    //  static navigationOptions = ({ this.props.navigate }) => ({
-    //     title: firstName + " " + lastName,
-    //      headerTitleStyle : {textAlign: 'center',alignSelf:'center'},
-    //         headerStyle:{
-    //             backgroundColor:'white',
-    //         },
-    //     });
-
-    pieColour(i) {
-
-    };
-
-    componentDidMount(){
-        //TODO: Remove hard coded dates/times once date picker works
-        // let startTime = new Date(Date.UTC(2018, 11, 11, 0, 0, 0, 0)).getTime();
-        // let endTime = new Date(Date.UTC(2018, 11, 11, 23, 0, 0, 0)).getTime();
-
-        let startTime = new Date(Date.UTC(2019, 1, 9, 0, 0, 0, 0)).getTime();
-        let endTime = new Date(Date.UTC(2019, 1, 9, 23, 0, 0, 0)).getTime();
-
-        const { navigation } = this.props;
-        const id = navigation.getParam('id');
+    getPatientData() {
+        var startDate = this.state.date;
+        var endDate = new Date(this.state.date);
+
+        switch(this.props.tabView) {
+            case Tabs.daily:
+                endDate.setDate(endDate.getDate() + 1);
+                break;
+            case Tabs.weekly: 
```

Comment:
With moment.js we could do `endDate.add(1, 'w');`

Take a look at this: https://momentjs.com/docs/#/manipulating/add/